Examples/Python/src/Geometry.cpp

```diff
@@ -67,6 +78,18 @@ void addGeometry(Context& ctx) {
         .value("Other", Acts::Surface::SurfaceType::Other);
   }
 
+  {
+    py::enum_<Acts::VolumeBounds::BoundsType>(m, "VolumeType")
```

Comment:
Nest this?

Reply:
This is just mirrored from the Surfaces:

```
  {
    py::enum_<Acts::Surface::SurfaceType>(m, "SurfaceType")
        .value("Cone", Acts::Surface::SurfaceType::Cone)
        .value("Cylinder", Acts::Surface::SurfaceType::Cylinder)
        .value("Disc", Acts::Surface::SurfaceType::Disc)
        .value("Perigee", Acts::Surface::SurfaceType::Perigee)
        .value("Plane", Acts::Surface::SurfaceType::Plane)
        .value("Straw", Acts::Surface::SurfaceType::Straw)
        .value("Curvilinear", Acts::Surface::SurfaceType::Curvilinear)
        .value("Other", Acts::Surface::SurfaceType::Other);
  }
```

Reply:
I suggest we make a decision how we do this consistently and then clean all of this out.

Reply:
Fair enough. I think we have `Surface` itself exposed as well, so nesting that enum would be easy I believe.